<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomLikeBluePaintProClass</class>
 <widget class="QWidget" name="CustomLikeBluePaintProClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CustomLikeBluePaintPro</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>701</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="lineToolButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>连线</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="outputEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="calculateButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="flashButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>更新关联</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CustomLikeBluePaintPro.qrc"/>
 </resources>
 <connections/>
</ui>
